.github/workflows/slow.yaml

```
@@ -197,11 +197,12 @@ jobs:
         uses: vmactions/openbsd-vm@v1
         with:
           usesh: true
+          release: "7.7"
           prepare: |
             pkg_add \
               autoconf-2.72p0 \
               autoconf-archive \
-              automake-1.16.5 \
+              automake-1.17 \
```

If comments are legal/OK here, please document the hidden dependency (missed in earlier PR commits). For example:

```suggestion
              automake-1.17 \ # keep in sync with AUTOMAKE_VERSION below
```

or perhaps

```suggestion
              automake-1.17 # keep in sync with AUTOMAKE_VERSION below \
```

I do not know which variant (if any) is correct in this context.

Alternatively, we may be able to define OpenBSD Automake version once and use it in both places, but I do not have a recipe for that. Our CHECKOUT_FETCH_DEPTH recipe would not work well here because Automake version differs across jobs.

---

Neither works, it's shell escaping.
And using a variable also doesn't work, the versions number might not align (e.g. 1.16.5 would correspond to 1.16).
I'll iterate on this solution, let's push it in as it is in order to unblock the other PRs

---

> I'll iterate on this solution

Please do.